src/index.test.ts

```diff
@@ -33,3 +33,54 @@ describe('getAllUsers', () => {
     expect(actual).toEqual(expected);
 })
 })
+
+describe('addUser', () => {
+  it('should create a unique id for an inputted user object and assign the id to the key of the user object in the users map', () => {
+    const userApi = new UserAPI();
+    const expected = userApi.getAllUsers().length + 1;
+    userApi.addUser(new_user);
+
+    const actual = userApi.getAllUsers().length;
+
+    expect(actual).toEqual(expected);
+  })
+
+  it('should return a string if the user is added successfully', () => {
+    const userApi = new UserAPI();
+    const expected = "User successfully added";
+
+    const actual = userApi.addUser(new_user);
+
+    expect(actual).toEqual(expected);
+  })
+})
+
+describe('getUserById', () => {
+  it('should take an id as a parameter and return a single object', () => {
+    const userApi = new UserAPI(user_repository);
+    const id = "564"
+    const expected = {
+      name: "Cadder Jenks",
+      age: 67,
+      favColor: "Barbie",
+      id: "564"
+    }
+
+    const actual = userApi.getUserById(id);
```

Comment:
```suggestion
    const actual = userApi.getUserById(expected.id);
```



use `expected.id` rather than `const id`.  Couple of reasons: less code, and it ensures that your code is using the correct data or values.

if your test involved testing mismatched id's, then `const fooID` would be necessary.

this is a super nitpick comment.  :)

Reply:
addressed